site/content/docs/main/contributions/oracle-config.md

```diff
@@ -40,8 +40,7 @@ cluster resources. It provides a simple, configurable, and operationally robust
 way to back up your application state and associated data.
 
 If you're familiar with kubectl, Velero supports a similar model, allowing you to
-execute commands such as 'velero get backup' and 'velero create schedule'. The same
-operations can also be performed as 'velero backup get' and 'velero schedule create'.
+execute commands such as 'velero backup get' and 'velero schedule create'.
```

Comment:
It looks like this part of the documentation is copied directly from the output of `velero help`: https://github.com/vmware-tanzu/velero/blob/main/pkg/cmd/velero/velero.go#L68-L70

That text will need to be updated. In fact, we may want to remove it as kubectl doesn't support commands in the format `kubectl <resource> <verb>`.

Reply:
nice catch!